README.md

```diff
@@ -1,3 +1,57 @@
 # REDCap to Harvest
 
 Dockerfile and tools for deriving Harvest applications from REDCap projects.
+
+Installation and Setup
+----------------------
+
+This tool runs using Docker. Install it if you don't have it:
+
+[Install Docker](http://docs.docker.com/installation/) 
+
+If you are using Mac, use boot2docker to run Docker.
+
+
+After docker is installed, enter the directory that contains this repository and build the Docker container:
+
+```bash
+docker build .
+```
+
+When the build is complete, find the image docker built:
+```bash
+docker images
+```
+You should see a list of images like this:
```

Comment:
Let's change the volume mount in the container to be `/usr/src/app/files` so it next to the `project` folder.